src/Integration.UnitTests/Telemetry/TelemetryDataRepositoryTests.cs

```diff
@@ -53,7 +53,7 @@ public void Ctor_Create_Storage_File()
             var repository = new TelemetryDataRepository(fileSystemMock.Object, watcherFactoryMock.Object);
 
             // Assert
-            RemoveLineEndings(fileContents.ToString()).Should().Be(RemoveLineEndings(@"<?xml version=""1.0"" encoding=""utf-16""?>
```

Comment:
The line endings were unnecessary as the test still fails when the tabs/spaces don't match. This actually made the test's failure much harder to read.